plugins/tff_backend/consts/hoster.py

```diff
@@ -24,7 +24,9 @@
 DEBUG_NODE_DATA = None
 if DEBUG:
     DEBUG_NODE_DATA = {
-        'status': {'status': 'running'},
+        'id': 'fake node id',
+        'serial_number': 'fake serial number',
+        'status': 'running',
```

Comment:
shouldnt this be `'status': {'status': 'running'},` ?

Reply:
nope